tests/coverage/async_closure.rs

```diff
@@ -4,7 +4,7 @@
 //@ aux-build: executor.rs
 extern crate executor;
 
-async fn call_once(f: impl async FnOnce()) {
```

Comment:
This will need to be re-blessed with `build.profiler = true` so that it doesn't fail in full CI.

(Thankfully that setting is compatible with `download-ci-llvm` nowadays, so it should be quick.)